### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Alex93Troy**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,13 +1,41 @@
...
+            "message": error["text"],
+            "name": error["code"],
+            "source": "flake8"}
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {
+        "errors": [
```
Comment (vasylhnatiuk) [inline app/main.py]: reuse format_linter_error here, don't duplicate your code

File: app/main.py [hunk 2]
```diff
@@ -1,13 +1,41 @@
...
 
 
 def format_linter_report(linter_report: dict) -> list:
-    # write your code here
-    pass
+    return [
+        {
+            "errors": [
+                {
+                    "line": error["line_number"],
```
Comment (vasylhnatiuk) [inline app/main.py]: reuse format_single_linter_file here

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 2 above)
Comment (Alex93Troy) [reply app/main.py]: Okay, thank you. I edit code without duplicate

Comment (Alex93Troy) [reply app/main.py]: Task is done or somthing wrong ?

**Pull request closed without merging**